_data/simple-icons.json

```
@@ -11975,6 +11975,11 @@
             "hex": "0099FF",
             "source": "https://www.threadless.com/about-us/"
         },
+        {
+            "title": "Threads",
+            "hex": "000000",
+            "source": "https://en.wikipedia.org/wiki/Threads_(app)#/media/File:Threads_(app)_logo.svg"
```

```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:Threads_(app)_logo.svg"
```

Please update the source to the Wikimedia Commons location 😄